src/pages/about/index.mdx

```diff
@@ -10,13 +10,15 @@ IBM.com is a place for people to learn about, try, and buy IBM products and serv
 
 <PageDescription>
 
-Towards that end, Carbon for IBM.com has been built with the goal of providing efficiency and consistency across IBM.com using IBM Design Language (IDL) principles and based on the Carbon Design System (Carbon).
+Towards that end, Carbon for IBM.com has been built with the goal of providing efficiency and consistency across IBM.com using IBM Design Language (IDL) principles and based on the Carbon Design System (Carbon).  
+
+This site is intended to provide guidance and resources for designers and developers who create sites and/or applications on IBM.com and other IBM domains.
 
 </PageDescription>
 
 <InlineNotification>
 
-**Note:** This site is intended to provide guidance and resources for designers and developers who create sites and/or applications on IBM.com and other IBM domains.
+**Note:** Carbon for IBM.com is the strategic design system for IBM web experiences.  Any application using older older systems like Northstar (v18), v19, or even earlier versions should make plans to adopt Carbon for ibm.com by the end of 2021.  After all applications have migrated, plans and dates will be announced for sunsetting the older systems.  Reach out to us in our IBM slack channel at [#carbon-for-ibm-dotcom](https://ibm-studios.slack.com/archives/C2PLX8GQ6) with any questions.    
 
```

Comment:
**Note:** Carbon for IBM.com is the strategic design system for IBM web experiences.  Any application using older systems like Northstar (v18), v19, or even earlier versions should make plans to adopt Carbon for ibm.com by the end of 2021.  After all applications have migrated, plans and dates will be announced for sunsetting the older systems.  Reach out to us in our IBM slack channel at [#carbon-for-ibm-dotcom](https://ibm-studios.slack.com/archives/C2PLX8GQ6) with any questions.